go.mod

```diff
@@ -1,112 +1,116 @@
 module sigs.k8s.io/prometheus-adapter
 
-go 1.18
+go 1.20
```

Comment:
this should probably be updated in the Makefile as well

Reply:
good catch :+1: